code/modules/clothing/under/ties.dm

```diff
@@ -763,7 +665,7 @@ obj/item/storage/internal/accessory/knifeharness/duelling
 		if(isgun(W))
 			if(current_gun)
 				if(!stop_messages)
-					to_chat(usr, SPAN_WARNING("[src] already holds a gun."))
+					to_chat(usr, SPAN_WARNING("[src] already holds \a [W]."))
```

Comment:
this message only comes up on attempted insertion so your CL is wrong

Reply:
Corrected the changelog to reflect actual behaviour change